### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by Malcolm-Shepherd**

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return (int)((double)numberCorrectAnswers / (double)numberOfQuestions * 100) + "%";
```
Comment (DaisyPalencia) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: For this you don't need to cast the line as an int, so I would recommend to remove the int from your code.

**Review by Wvvern — COMMENTED** [review]
- Pull Request title includes "Assignment 1" in the title ✔
- Pull Request targets Assignment 1 branch ✔
- Issue 1: Application no longer crashes ✔
- Issue 2: Unit test properly passes ✔

**Review by DaisyPalencia — COMMENTED** [review]
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs (hunk 1 above)
Comment (JLee35) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs]: The 'int' cast actually makes the result cleaner for the user ('85%' versus '85.67547%'), so I think it is a great addition.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 2]
```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return ((double)numberCorrectAnswers / (double)numberOfQuestions * 100) + "%";
```
Comment (JLee35) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: Casting to an `int` or using `math.round` would have cleaned this up for the user.

**Comment by JLee35** [comment]
Ensure to address review comments next time.

**Pull request closed without merging**